content/kubermatic/master/tutorials_howtos/administration/admin_panel/_index.en.md

```diff
@@ -4,42 +4,106 @@ date = 2018-08-09T12:07:15+02:00
 weight = 40
 +++
 
-The Admin Panel is a place for the Kubermatic administrators where they can manage the global settings that directly
-impact all Kubermatic users. Admin rights can be granted from the admin panel itself and also from the kubectl by
+The Admin Panel is a section where the Kubermatic administrators can manage the global settings that
```

Comment:
Why changing "place" to "section"?

Reply:
Because a place looks awkward in the context since it is a doc, not a location or physical structure. However, I am thinking of using it like this:  "The Admin Panel is "where" the Kubermatic administrators................"

Reply:
Yes, but this is not related to the doc but to the place/view in the app.

Reply:
Alright. I can change it if it suits better